vizro-core/docs/pages/user-guides/install.md

```diff
@@ -1,50 +1,140 @@
-# How to install Vizro
+# Installation Guide for Vizro
 
-Thanks to [PyCafe](https://py.cafe/) you can create Vizro dashboards without installing Vizro locally, as you can see from the [first dashboard tutorial](../tutorials/first-dashboard.md) (you can find out more about [working with PyCafe and Vizro](https://py.cafe/docs/apps/vizro) from the PyCafe documentation).
+## Overview
 
-If you prefer to work locally, this guide shows you how to install Vizro, how to verify the installation succeeded and find the version of Vizro, and how to update Vizro.
+This guide provides a comprehensive overview of how to install Vizro, a tool for creating interactive dashboards. You can use Vizro without local installation via [PyCafe](https://py.cafe/), as demonstrated in the [first dashboard tutorial](../tutorials/first-dashboard.md). For further details on integrating PyCafe with Vizro, refer to the [PyCafe documentation](https://py.cafe/docs/apps/vizro).
```

Comment:
This is a bit confusing now there's a mix of mentions of PyCafe and local installation. It's best for the cognitive load to keep things separate. Let me try an suggestion...